scripts/src/java/org/oppia/android/scripts/XMLSyntaxCheck.kt

```diff
@@ -0,0 +1,71 @@
+package org.oppia.android.scripts
+
+import org.xml.sax.SAXParseException
+import java.io.File
+import javax.xml.parsers.DocumentBuilder
+import javax.xml.parsers.DocumentBuilderFactory
+
+/**
+ * Script for ensuring that all the XML files
+ * in the repo are syntactically correct.
+ */
+class XMLSyntaxCheck {
+  companion object {
+    @JvmStatic
+    fun main(vararg args: String) {
+      // path of the repo to be analyzed.
```

Comment:
Please fix single line comment casing & line wrapping per earlier comments here & in #3340.

Reply:
Done.